package.json

```diff
@@ -63,6 +63,7 @@
     "xmlhttprequest": "^1.8.0"
   },
   "dependencies": {
-    "tweetnacl": "^1.0.3"
+    "tweetnacl": "^1.0.3",
+    "@types/node": "^14.14.31"
```

Comment:
shouldn't that be a dev dependency ? and if it stays a regular dependency, shouldn't it allow a wider range ?